infoschema/metricschema/table.go

```diff
@@ -64,12 +68,12 @@ func GetMetricTableDef(lowerTableName string) (*MetricTableDef, error) {
 }
 
 // GetExplainInfo uses to get the explain info of metric table.
-func GetExplainInfo(sctx sessionctx.Context, lowerTableName string) string {
+func GetExplainInfo(sctx sessionctx.Context, lowerTableName string, labels map[string]set.StringSet, quantile float64) string {
```

Comment:
Why not place this method to `PhysicalMemTable`?

Reply:
Already remove this, no need it any more.